eZ/Publish/SPI/Repository/Values/MultiLanguageDescriptionValueAware.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\SPI\Repository\Values;
+
+/**
+ * This is the interface for all ValueObjects implementing translated description logic.
+ *
+ * Provides a uniform way for API consuming logic to generate translated description labels
+ * for API objects.
+ */
+interface MultiLanguageDescriptionValueAware
```

Comment:
Sure. I made `...Aware` name on the fly, just to point we need to rename it from `...Base`.